src/components/IconButton.tsx

```diff
@@ -21,6 +21,7 @@ export const IconButton = ({
   return (
     <Container accessibilityLabel={accessibilityLabel} {...otherProps}>
       <SvgXml
+        testID="icon-button-svg"
```

Comment:
this felt weird but eventually made some sense since this is a presentational component and it's logic is based on proxying the props down, happy to remove this.

Reply:
Also, this file is using some form of styled component that I'm not sure we need, thoughts?

Reply:
yup, removed in another commit!  Also worth noting that this component is only used in one spot. What do you think about inlining it?

Reply:
This is only used on the `Eula` component, I think that adding more into that file might have to be reverted later, happy to do it if we feel is better

Reply:
@johnschoeman I moved the components into the Eula Modal and transformed it to Typescript, you mentioned something about letting the GPS team know, should I point them back to this commit?

Reply:
Yes, or if you are feeling particularly generous you could open a similar commit to the GPS repo.